### ISSUE TITLE
(Feature request) Make interface auth timeout configurable

### ISSUE THREAD

**Issue by BeecherNetworks** [zadam/trilium#2722]
Could this be added on the Other tab in Options, near the other timeouts? I'd guess I'm not the only person using Trilium just for myself, it would be nice to stay logged in for days / weeks / etc. while retaining password protection.

_Originally posted by @BeecherNetworks in https://github.com/zadam/trilium/discussions/2720_

**Comment by dahamsta** [comment]
Yes, I just keep a pinned tab open in Firefox. That's set to 600 for me, so I don't think it's that that's booting me.

**Comment by dahamsta** [comment]
Fair point, I'll have a look, thanks.

**Comment by eliandoran** [comment]
This feature exists now: https://docs.triliumnotes.org/user-guide/setup/server/auth#understanding-how-the-session-works
